buddyboss/buddyboss-platform
# Pasting content with medium editor

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2613

### How to test the changes in this Pull Request:

1. Make sure to only activate platform plugin
2. Make also to deactivate platform pro
3. Go to newsfeed and try pasting url using ctrl + v
4. See error


### Other information:
